File: ex4.cpp
#include <iostream>
using namespace std;
int main()
{
	int a, b;
	cin >> a;
    b = 0;
	while  (a != 0)
	{
      if (a % 3 == 0) b = b + 1;
      cin >> a;
    }
    cout << b;	
}

File: Task6/ex4.py
#include<iostream>
using namespace std;
int main() 
{ 
    int a,b,c;
    cin >> a >> b >> c;
    if((a+b>c)&&(a+c>b)&&(b+c>a)) 
     {
         if(a==b&&b==c) cout<<"Equilateral";
         else if (a==b||a==c||b==c) cout<<"Isosceles";
              else cout<<"Scalene";
     }
    else cout<<"Invalid";
}